content/pages/6.about.rst

```diff
@@ -43,36 +35,13 @@ I will
 * Never be condescending.
 * Never discriminate (even if you like Emacs and Java).
 
-If I ever fail one of the tenets above, please let me know. After all, I am
-human.
-
 Exception
 =========
 
 There is one exception. 
 
-I don't react well when someone is being aggressive or insulting. I will
-politely let you know that you're being unpleasant and I will stop responding.
-
-Caveats
-=======
-
-1. English is not my native tongue. 
-
-2. I don't fully grasp most colloquial slang, but it never stops me from using
-   it ;).  
-   
-   Here's an example, until very recently I thought 'pretty good' meant 'very
-   good'. So I would compliment my friends that they did a 'pretty good' job
-   without knowing that I was mildly insulting them. I'm told 'pretty good'
-   means not quite good enough or there is room for improvement.
-
-3. I think I'm funny (no, I think I'm hilarious). Sometimes that might come
-   across negatively without proper context. If anything I say offends you,
-   please let me know.
-
-But this is not a cop out, I'd still appreciate if you call me out when I make
-a faux pas. 
+If you're being aggressive or insulting we will politely let you know that
+you're being unpleasant and I will stop responding.
```

Comment:
Maybe like this?

> If you're being aggressive or insulting we will politely let you know that
you're being unpleasant. If you persist, we will stop responding.